libraries/terminal/src/ConsoleTerminalProvider.ts

```diff
@@ -27,11 +27,13 @@ export interface IConsoleTerminalProviderOptions {
 
```

Comment:
Line 5 still says:
```ts
import { enabled as supportsColor } from 'colors/safe';
```

...so we should make sure that `colors` is still a declared regular dependency of `@rushstack/terminal` (unless we can eventually find some other implementation of `enabled`).

Reply:
Yeah I want to replace that eventually, but it is still a normal dependency.